src/axom/quest/MarchingCubes.hpp

```diff
@@ -37,7 +37,8 @@ namespace detail
 namespace marching_cubes
 {
 template <int DIM, typename ExecSpace, typename SequentialLoopPolicy>
-class MarchingCubesImpl;
+class MarchingCubesImpl;  // TODO: Delete this.
```

Comment:
Does deleting this line go with this PR, or is that saved for future work?

Reply:
Sorry.  It was suppose to have been deleted.